### PR TITLE
Added Angle Arrow Detector Project.

### LINKED ISSUE #263: Detecting any arrow from a live camera feed.
# Description

A python project which uses OpenCV library to detect any arrow shaped object, along with its directions, from a live camera feed. This could have many useful applications like detecting traffic signs and to provide instructions for navigation.

## Type of issue
- NOTE: *These boxes can be checked using **`[X]`***

- [x] Feature (New Script)
- [ ] Bug
- [ ] Documentation

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by VyomGarg47**
# Description

Added Arrow detector project which detects any arrow, along with its directions, from a live camera feed.
Fixes #263 
Replace `issue_no` in the above line, with the issue related to this PR.
## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Miscellaneous/Arrow_Angle_detector/Arrow.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+						y = approx.ravel()[1]
+						
+						#x1 = n[0]
+						#y1 = n[1]
+						#x2 = n[2]
+						#y2 = n[3]
+						#x3 = n[6]
+						#y3 = n[7]
+						#distance1 = (x1 - x2)*(x1 - x2) + (y1 - y2)*(y1 - y2)
+						#distance2 = (x1 - x3)*(x1 - x3) + (y1 - y3)*(y1 - y3)
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/Arrow_Angle_detector/Arrow.py:57]: Remove unnecessary commented out code lines, add comments instead for better understanding,

File: Scripts/Miscellaneous/Arrow_Angle_detector/Arrow.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import cv2
+import numpy as np
+
+
+def nothing(x):
+	pass
+
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/Arrow_Angle_detector/Arrow.py:7]: Please remove this.

**Review by tusharnankani — CHANGES_REQUESTED** [review]
- Please follow this [README TEMPLATE.](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)

**Comment by VyomGarg47** [comment]
I have made the necessary changes. Kindly review.

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Break your code into functions, where each function performs a specific task, in order to improve the readability of the code.

**Comment by tusharnankani** [comment]
> Break your code into functions, where each function performs a specific task, in order to improve the readability of the code.

Are you working on it? @VyomGarg47

**Comment by VyomGarg47** [comment]
@tusharnankani I'm extremely sorry for the delay. Kindly review again.

**Review by tusharnankani — APPROVED** [review]
Looks good to me! 👍🏻

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@allcotributors please add @VyomGarg47 for code

**Comment by AdityaJ7** [comment]
@allcontributors please add @VyomGarg47 for code

**Pull request merged**